packages/cli/src/cog/builder.ts

```diff
@@ -226,6 +226,7 @@ export class CogBuilder {
       else union = Bounds.fromJson(bounds).union(union);
     }
     if (union == null) throw new Error('Bug! union can not be null');
+    if (existsSync(CacheFolder)) rmdirSync(CacheFolder, { recursive: true });
```

Comment:
This defeats the purpose of having a cache.

the goal was to cache things between runs. can we disable the cache instead?

Reply:
What do you mean disable cache? Just create meta data on fly?

Reply:
this cache folder was so that if you run the command twice you can use the cached data

eg
```
./import s3://a
./import s3://a --commit
```

So it doesnt have to recompute everything

If we delete the cache after the first import invocation there is no point to having a cache.

so rather than just deleting the cache folder it would be better just to stop using the cache.

Reply:
I see what you mean. Do we want to make this as a option parameter? For the import api, it seems we always creating new job if there is any changes. But we can still using this for import cli?

Reply:
If we are moving this all into the cloud then lets just delete it, not point storing the cache here.

Reply:
All fixed. Thank you.